CMakeLists.txt

```diff
@@ -70,17 +72,28 @@ file(GLOB TINYGETTEXT_HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} include/tinyg
 
 add_library(tinygettext ${TINYGETTEXT_SOURCES})
 
+## Fix Windows exports
+
+INCLUDE (GenerateExportHeader)
+GENERATE_EXPORT_HEADER( tinygettext
+             BASE_NAME tinygettext
+             EXPORT_MACRO_NAME TINYGETTEXT_API
+             EXPORT_FILE_NAME tinygettext_Export.h
+             STATIC_DEFINE tinygettext_BUILT_AS_STATIC
+)
+
+
 ## Add tinygettext dir to search path
-target_include_directories(tinygettext PUBLIC include/)
+target_include_directories(tinygettext PUBLIC include/ ${CMAKE_CURRENT_BINARY_DIR})
 
 ## SDL/iconv usage
 option(HAVE_SDL "Use SDL_iconv instead of system iconv" FALSE)
 if(HAVE_SDL)
 set_source_files_properties(${TINYGETTEXT_SOURCES} PROPERTIES COMPILE_DEFINITIONS HAVE_SDL)
-TARGET_INCLUDE_DIRECTORIES(tinygettext PUBLIC ${SDL2_INCLUDE_DIRS})
+INCLUDE_DIRECTORIES(tinygettext PUBLIC ${SDL2_INCLUDE_DIRS})
 TARGET_LINK_LIBRARIES(tinygettext PUBLIC ${SDL2_LIBRARIES})
 
-else(HAVE_SDL)
+endif(HAVE_SDL)
```

Comment:
I will re-work this pull request and resubmit it.  This was actually done (apparently) because when it attempts to build the tests, HAVE_SDL is unset, forcing it to try to use iconv.  I believe that was my reasoning for the TARGET_INCLUDE_DIRECTORIES thing as well.